engine/src/main/java/org/camunda/bpm/engine/impl/cmd/GetTelemetryDataCmd.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.impl.cmd;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.camunda.bpm.engine.impl.interceptor.Command;
+import org.camunda.bpm.engine.impl.interceptor.CommandContext;
+import org.camunda.bpm.engine.impl.metrics.Meter;
+import org.camunda.bpm.engine.impl.metrics.MetricsRegistry;
+import org.camunda.bpm.engine.impl.telemetry.CommandCounter;
+import org.camunda.bpm.engine.impl.telemetry.TelemetryRegistry;
+import org.camunda.bpm.engine.impl.telemetry.dto.CommandImpl;
+import org.camunda.bpm.engine.impl.telemetry.dto.InternalsImpl;
+import org.camunda.bpm.engine.impl.telemetry.dto.MetricImpl;
+import org.camunda.bpm.engine.impl.telemetry.dto.TelemetryDataImpl;
+import org.camunda.bpm.engine.impl.telemetry.reporter.TelemetrySendingTask;
+import org.camunda.bpm.engine.telemetry.dto.Metric;
+
+public class GetTelemetryDataCmd implements Command<TelemetryDataImpl> {
+
+  ProcessEngineConfigurationImpl configuration;
+  TelemetryRegistry telemetryRegistry;
+  MetricsRegistry metricsRegistry;
+
+  @Override
+  public TelemetryDataImpl execute(CommandContext commandContext) {
+    configuration = commandContext.getProcessEngineConfiguration();
+    telemetryRegistry = configuration.getTelemetryRegistry();
+    metricsRegistry = configuration.getMetricsRegistry();
+
+    TelemetryDataImpl telemetryData = commandContext.getProcessEngineConfiguration().getTelemetryData();
+    InternalsImpl internalsDynamicData = new InternalsImpl();
+    internalsDynamicData.setCommands(getCommandCounts());
+    internalsDynamicData.setMetrics(getMetrics());
+    telemetryData.getProduct().getInternals().mergeDynamicData(internalsDynamicData);
```

Comment:
This codes duplicates the logic in `TelemetrySendTask#run` (https://github.com/camunda/camunda-bpm-platform/blob/master/engine/src/main/java/org/camunda/bpm/engine/impl/telemetry/reporter/TelemetrySendingTask.java#L123-L126). I think this has some issues:

* It doesn't update the telemetry data with what is collected in the telemetry registry after the engine is started (e.g. application server and webapp usage); This will therefore miss in setups with disabled telemetry reporting
* The duplicated code introduces the possibility for future bugs (e.g. how metrics are collected is implemented in two places)
* A potential bug: The metrics and commands are now stored in the object that is held in the process engine configuration and that is reused by the `TelemetrySendTask`. I don't think this leads to a bug as of now, but is at least a code smell because it leaks the temporary data of the `GetTelemetryDataCmd` into a "persistent" object

I suggest we implement a solution that reuses the existing logic.